README.md

```diff
@@ -148,16 +150,16 @@ if (biometrySettingsChanged && usingBiometry) {
 
 ```swift
 // Swift
-let biometrySettingsChanged = Locker.biometricsSettingsAreChanged
+let biometrySettingsChanged = Locker.biometricsSettingsDidChange
 let usingBiometry = Locker.shouldUseAuthenticationWithBiometrics(for: "kUniqueIdentifier")
 if biometrySettingsChanged && usingBiometry {
 // handle case when settings are changed and biometry should be used
 }
 ```
 
-##### 7. There are `deviceSupportsAuthenticationWithBiometrics` and `canUseAuthenticationWithBiometrics` methods which return `BiometricsType` enum (`BiometricsTypeNone`, `BiometricsTypeTouchID`, `BiometricsTypeFaceID`).
-`deviceSupportsAuthenticationWithBiometrics` checks if device has support for some Biometric type.
-`canUseAuthenticationWithBiometrics` checks if device has support for some Biometrics type and if that Biometric is enabled in device settings.
+##### 7. There are `supportedBiometricsAuthentication` and `configuredBiometricsAuthentication` computed properties which return `BiometricsType` enum (`BiometricsTypeNone`, `BiometricsTypeTouchID`, `BiometricsTypeFaceID`).
+`supportedBiometricsAuthentication` checks if device has support for some Biometric type.
+`configuredBiometricsAuthentication` checks if device has support for some Biometrics type and if that Biometric is enabled in device settings.
```

Comment:
Please write down (somewhere, notes, task, ...) everything that you changed in the readme. 
Upon release, we have to write "changelog" with all property/method name changes. That way, the user will know which old method should be renamed to which new name.

Reply:
Wrote down in notes when I changed the readme back to its previous state on the apiary branch.